### ISSUE TITLE
SCM Viewlet should be more keyboard friendly

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#90893]
In short: I should be able to navigate in the SCM viewlet and press a key to stage / commit a change and the focus should remain in the SCM viewlet.
Enter currently opens the change and moves focus. I think that is fine, but we could have a shift + enter or something like that for stage / commit and keep focus.

A bit longer from @ndarilek

Sometimes I make several sets of changes, and want
to split them into logical commits. It can be hard to limit a set of
changes to a single logicl commit because often they need to be tested
together, and one won't make much sense if another is impractical or
unneeded. Right now I:

1. Press Ctrl-Shift-G to jump to the VCS changes view.

2. Tab to my changes.

3. Find a change in the tree.

4. Press the Applications key.

5. Arrow down to Stage Changes.

6. Press Enter.

7. Repeat 4-6 a few times...

Is it possible to stage changes just by pressing Enter on an item? That,
to me, seems most logical, with Discard and other operations remaining
reachable through the context menu. More often than not, I'm in this
view wanting to stage changes.

Yes I can set up Smart Commit to stage everything, but I've already
accidentally committed secrets that way. :) Yes I can group untracked
files and not Smart Commit those, but that took a bit of digging to set
up. I just think that the default in this view will almost always be to
stage the currently-selected change, so Enter should do that, though I
certainly won't bikeshed it if there are good reasons not to do this.
Just felt like I had to dig a git to find this, and VSCode nudged me
toward a behavior that, in its default configuration, probably isn't
wise (at the very least, Smart Commit shouldn't default to also adding
untracked files.)

**Comment by joaomoreno** [comment]
The contextual actions in each row can also be focused by pressing <kbd>Tab</kbd> and then <kbd>Enter</kbd> to execute. The only thing that I see missing is the fact that the SCM view loses focus once that happens. I can push a fix for that.

Another thing I've decided to push is a <kbd>Ctrl Enter</kbd> keybinding for lists which toggles the currently focused item from the selection. You could already kind of doing this with <kbd>Shift Up/Down</kbd>, but that would work only for consecutive elements. With <kbd>Ctrl Enter</kbd>, you can now select each item to be staged and then execute the action only once.

![Peek 2020-02-19 09-37](https://user-images.githubusercontent.com/22350/74816542-a8c56880-52fb-11ea-9742-0632b76fc5e7.gif)

**Comment by isidorn** [comment]
Thanks for pushing a fix for this. Just tried to verify it but the insiders from 5 mins ago does not contain the fix. Will try it tomorrow.

**Comment by jvesouza** [comment]
A problem I found is that using orca there is no return indicating that the file is selected or not when I press ctrl + shift + enter.
Perhaps this problem is related to the orca. I cannot test with another screen reader.

**Comment by isidorn** [comment]
Adding verified label since this works as expected.
@jvesouza you are correct, this is missing. For this I have filed this issue https://github.com/microsoft/vscode/issues/91061

**Comment by jvesouza** [comment]
Another problem using orca is that when I browse the list, orca does not announce the selected items, if one is selected.

**Comment by isidorn** [comment]
@jvesouza that sounds like an Orca issue. Can you maybe send a mail to the orca list and ask them the following:
* VS Code sets the `aria-selected: true` and Orca does not announce that the item is selected. What is missing here?

**Comment by jvesouza** [comment]
@isidorn I'll check, thanks.
I found on more problem using orca:
Steps to reproduce:

* In the list of changes use the arrows to choose a change.
* Press the application key.
* Choose an operation using the arrows.
* Press the escape key.
* Try to navigate in the list using the arrows.

On my machine the orca is muted and I need to press tab and shift + tab to get the orca to respond normally.

**Comment by isidorn** [comment]
@jvesouza thanks, I can reproduce that. For that I filled a new issue https://github.com/microsoft/vscode/issues/91085

**Comment by jvesouza** [comment]
If instead of pressing the escape key I press the enter key to accept 
the chosen operation, the problem also happens.

**Comment by isidorn** [comment]
Well the focus is left in the SCM view, it just jumps to the Tree container. Idealy the focus should be left in the same section of the view. @joaomoreno
